tests/integrationv2/common.py

```diff
@@ -33,7 +33,7 @@ def pq_enabled():
     """
     Returns true or false to indicate whether PQ crypto is enabled in s2n
     """
-    return not (get_flag(S2N_NO_PQ, False) or get_flag(S2N_FIPS_MODE, False))
+    return "awslc" in get_flag(S2N_PROVIDER_VERSION)
```

Comment:
this is true for newer versions of AWS-LC, but [not versions < 1.4.0](https://github.com/aws/s2n-tls/commit/10ec83aab502efae6a6f420d7e8b5e714ce3f155#diff-799d0dbdf0e2f812a8349e43aeda3cb520aedfc89a2e5c1d058b0d22e893efe1R64). should we include that version check here?

Reply:
I don't think we test with any <1.4.0 versions in the integration tests. Not sure we need to include that.

Reply:
ah, right. that makes sense.